--- ../berkelium/win32/subprocess/subprocess.vcproj	Thu Oct 27 16:41:49 2011
+++ ../berkelium/win32/subprocess/subprocess.vcproj	Wed Sep 12 15:20:53 2012
@@ -41,7 +41,7 @@
 			<Tool
 				Name="VCCLCompilerTool"
 				Optimization="0"
-				AdditionalIncludeDirectories="..\..\include;..\..\..\chromium\chromium\src;..\..\..\..\dependencies\ch\c\src"
+				AdditionalIncludeDirectories="..\..\include;..\..\..\..\chromium\chromium\src;..\..\..\..\dependencies\ch\c\src"
 				PreprocessorDefinitions="WIN32;_DEBUG;_WINDOWS"
 				MinimalRebuild="true"
 				BasicRuntimeChecks="2"
@@ -57,7 +57,7 @@
 			/>
 			<Tool
 				Name="VCResourceCompilerTool"
-				AdditionalIncludeDirectories="&quot;..\..\..\chromium\chromium\src\chrome\$(ConfigurationName)\obj\global_intermediate\chrome&quot;"
+				AdditionalIncludeDirectories="&quot;..\..\..\..\chromium\chromium\src\chrome\$(ConfigurationName)\obj\global_intermediate\chrome&quot;"
 			/>
 			<Tool
 				Name="VCPreLinkEventTool"
@@ -67,7 +67,7 @@
 				AdditionalDependencies="wininet.lib version.lib msimg32.lib ws2_32.lib usp10.lib psapi.lib dbghelp.lib urlmon.lib ..\..\bin\berkelium_d.lib sandbox.lib base.lib"
 				OutputFile="..\..\bin\berkelium_d.exe"
 				LinkIncremental="2"
-				AdditionalLibraryDirectories="..;..\..\..\chromium\chromium\src\chrome\Debug\lib;..\..\..\..\dependencies\ch\c\src\chrome\Debug\lib"
+				AdditionalLibraryDirectories="..;..\..\..\..\chromium\chromium\src\chrome\Debug\lib;..\..\..\..\dependencies\ch\c\src\chrome\Debug\lib"
 				IgnoreDefaultLibraryNames=""
 				DelayLoadDLLs=""
 				GenerateDebugInformation="true"
@@ -125,7 +125,7 @@
 				Name="VCCLCompilerTool"
 				Optimization="2"
 				EnableIntrinsicFunctions="true"
-				AdditionalIncludeDirectories="..\..\include;..\..\..\chromium\chromium\src;..\..\..\..\dependencies\ch\c\src"
+				AdditionalIncludeDirectories="..\..\include;..\..\..\..\chromium\chromium\src;..\..\..\..\dependencies\ch\c\src"
 				PreprocessorDefinitions="WIN32;NDEBUG;_WINDOWS"
 				RuntimeLibrary="0"
 				EnableFunctionLevelLinking="true"
@@ -138,7 +138,7 @@
 			/>
 			<Tool
 				Name="VCResourceCompilerTool"
-				AdditionalIncludeDirectories="&quot;..\..\..\chromium\chromium\src\chrome\$(ConfigurationName)\obj\global_intermediate\chrome&quot;"
+				AdditionalIncludeDirectories="&quot;..\..\..\..\chromium\chromium\src\chrome\$(ConfigurationName)\obj\global_intermediate\chrome&quot;"
 			/>
 			<Tool
 				Name="VCPreLinkEventTool"
@@ -149,7 +149,7 @@
 				AdditionalDependencies="wininet.lib version.lib msimg32.lib ws2_32.lib usp10.lib psapi.lib dbghelp.lib urlmon.lib berkelium.lib sandbox.lib base.lib"
 				OutputFile="..\..\bin\berkelium.exe"
 				LinkIncremental="1"
-				AdditionalLibraryDirectories="..;..\..\..\chromium\chromium\src\chrome\Release\lib;..\..\..\..\dependencies\ch\c\src\chrome\Release\lib;..\..\bin\"
+				AdditionalLibraryDirectories="..;..\..\..\..\chromium\chromium\src\chrome\Release\lib;..\..\..\..\dependencies\ch\c\src\chrome\Release\lib;..\..\bin\"
 				IgnoreDefaultLibraryNames=""
 				DelayLoadDLLs=""
 				GenerateDebugInformation="true"
@@ -208,7 +208,7 @@
 			UniqueIdentifier="{67DA6AB6-F800-4c08-8B7A-83BB121AAD01}"
 			>
 			<File
-				RelativePath="..\..\..\chromium\chromium\src\chrome\app\chrome_exe.rc"
+				RelativePath="..\..\..\..\chromium\chromium\src\chrome\app\chrome_exe.rc"
 				>
 			</File>
 		</Filter>
